src/Pages/Category/Category.js

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useEffect } from 'react';
+import { useLocation } from 'react-router';
+import styled from 'styled-components';
+import CategoryCard from '../../Components/CategoryCard/CategoryCard';
+import ExpertCard from '../../Components/ExpertCard/ExpertCard';
+import FilterDropdown from '../../Components/FilterDropdown/FilterDropdown';
+
+function Category(props) {
+  const [categoryData, setCategoryData] = useState([]);
+  const [expertData, setExpertData] = useState([]);
+  const [chooseCardFilter, setChooseCardFilter] = useState(0);
+  const [selectedSort, setSelectedSort] = useState('-sell_count');
+  const [cardCount, setCardCount] = useState(4);
+
+  const topFilterList = ['상담', '엑스퍼트'];
+  const category = {
+    1: 'JavaScript',
+    2: 'Python',
+    3: 'React',
+    4: 'HTML',
+    5: 'Go',
+    6: 'MySql',
+    7: 'Java',
+    8: 'C',
+    9: 'C++',
+  };
+
+  const CARD_FILTER =
+    chooseCardFilter === 0 ? 'products?category' : 'products/expert?category';
+
+  const makeBorderBottom = event => {
+    setChooseCardFilter(event.target.value);
+  };
+
+  const location = useLocation();
+
+  useEffect(() => {}, [selectedSort]);
+
+  useEffect(() => {
+    fetch(
+      `http://3.133.12.85:8000/products?category=${props.match.params.id}&sort=-sell_count`
+    )
+      .then(res => res.json())
+      .then(data => setCategoryData(data));
+  }, []);
+
+  useEffect(() => {
+    fetch(
+      `http://3.133.12.85:8000/products/expert?category=${props.match.params.id}&sort=-sell_count`
+    )
+      .then(res => res.json())
+      .then(data => setExpertData(data));
+  }, []);
```

Comment:
1. 같은 CDM 의 효과이기 때문에, 하나의 useEffect 안에서 호출해주세요!
```suggestion
  useEffect(() => {
    fetch(
      `http://3.133.12.85:8000/products?category=${props.match.params.id}&sort=-sell_count`
    )
      .then(res => res.json())
      .then(data => setCategoryData(data));
      
     fetch(
      `http://3.133.12.85:8000/products/expert?category=${props.match.params.id}&sort=-sell_count`
    )
      .then(res => res.json())
      .then(data => setExpertData(data));
      
  }, []);
```
2. 모든 fetch 가 완료된 후에, 일괄적으로 setState 를 하는 것이 더 적절할 것 같습니다. 이를 위해선 Promise.all 메서드를 사용해 보시면 좋을 것 같습니다. 아래 링크 참고해서 적용할 수 있으시면 적용 해보시고, 좀 어렵다 하시면 이런 메서드가 있다는 것만 알아주시고 후에 적용해보세요!
https://ko.javascript.info/promise-api
3. 마지막으로 API 부는 config에서 일괄적으로 관리해주세요!
```jsx
fetch(`${GET_PRODUCTS_API}?cateogroy .......`)
```